### DISCUSSION TITLE
MPD Won't start?

### DISCUSSION THREAD

**Discussion by margaretjoanmiller** [MusicPlayerDaemon/MPD#1488] [Q&A]
Hi! I'm having a strange issue with mpd. My config file appears to be valid but when I run mpd -v I get a "config_file: loading file /home/maeday/.config/mpd/mpd.conf" message and nothing else but mpc can't connect to the server and running pgrep mpd yields no results. It seems like mpd isn't running but I'm not seeing any errors. Here's a paste link for my mpd --version log and config file. Any help would be greatly appreciated!

https://paste.debian.net/1234760/

**Comment by MaxKellermann** [comment]
Your MPD version is very old and unsupported.

**Reply by margaretjoanmiller** [reply]
Would 0.23.5 be new enough? That's the newest version the distro I'm using (gentoo) has in its repositories (in a testing state). If not I can use the latest version manually.

**Reply by MaxKellermann** [reply]
Yes, 0.23.5 would be acceptable. Also check https://www.musicpd.org/help/

**Reply by margaretjoanmiller** [reply]
Okay cool. Also regarding the help page, I thought I included all of the things asked for on that page in my paste.debian.net link? Did I miss something. Also, I upgraded to 0.23.5 and the same issue is happening with the same output from `mpd -v`

**Reply by MaxKellermann** [reply]
You did not paste your log.
